Dockerfile

```diff
@@ -28,4 +27,4 @@ COPY . $APP_HOME
 
 EXPOSE 8000
 
-CMD /bin/sh -c "wait-for-it.sh $DATABASE_HOST:${DATABASE_PORT:-5432} -- ./manage.py migrate && uwsgi"
+CMD /bin/sh -c "wait-for-it.sh $DATABASE_HOST:${DATABASE_PORT:-5432} -- ./manage.py migrate && gunicorn --workers 10 --access-logfile - --limit-request-line 16384 --bind :8000 emeis.wsgi"
```

Comment:
We should pull `wait-for-it` from the stable release branch (apt), instead of cloning the latest master. This has nothing to do with the actual gunicorn migration but just poppend into my :eyes:.

Reply:
@c0rydoras This is a small, but important change. IMO you can add a commit to this PR for that.

Reply:
@c0rydoras This also needs a change on all calling sites. `'s/wait-for-it.sh/wait-for-it/g'`

Reply:
done